src/theme/components/accordion.ts

```diff
@@ -1,21 +1,28 @@
 const containerStyles = {
   borderLeftWidth: "1px",
   borderRightWidth: "1px",
-  borderColor: "ui.gray.medium",
   width: "100%",
+  _hover: {
+    borderLeftWidth: "1px",
+    borderRightWidth: "1px",
+    borderColor: "ui.gray.dark",
```

Comment:
Ticket 862 says the border should be `ui.gray.medium` on hover.

Reply:
The Accordion button border is `ui.gray.medium` on default, so if I add it on hover, there is no change, visually. The Figma file shows `ui.gray.dark` for hover and `ui.gray.medium` for default: https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=10734%3A2520. @bigfishdesign13